global/php/navbar.php

```diff
@@ -198,12 +237,20 @@
 
             <div class="osekai__apps-dropdown-applist-left-top">
                 <div class="osekai__apps-dropdown-applist-left-bottom" onclick="hideOtherApps()">
-                    Back to Apps
+                    <p><i class="fas fa-chevron-left"></i> back to apps</p>
                 </div>
                 <div class="osekai__apps-dropdown-other-content">
-                    <a class="osekai__apps-dropdown-other-content-button" href="/misc/translators/">
-                        Translators
-                    </a>
+                    <?php
+                    foreach ($otherApps as $oapp) {
+                        // would name this $app but for whatever fucking reason php decides that it's
+                        // a great idea to replace the variable "$app" with it and never put it 
+                        // back globally across the entire project if you dare to do that so we
+                        // have to kinda just be careful not to?? i hate this
+                        echo "<a class=\"osekai__apps-dropdown-other-content-button\" href=\"{$oapp['link']}\">
```

Comment:
imo would be better to use single quote marks here so the escaping of " is not needed, just a bit more readable

Reply:
last i checked those don't work in html?

Reply:
i mean the ones the echo is using, not the ones that are given for the class or href

Reply:
then you can't use {$variable}, that only works with double quotes

Reply:
my bad then, im still kinda new to php stuff